scripts/updates/utils/termux_pkg_upgrade_version.sh

```diff
@@ -67,9 +70,13 @@ termux_pkg_upgrade_version() {
 
 	# Translate "-suffix" into "~suffix": "X.Y.Z-suffix" is considered later
 	# than X.Y.Z. for it to be considered earlier use "X.Y.Z~suffix".
-	LATEST_VERSION="${LATEST_VERSION//-rc/~rc}"
-	LATEST_VERSION="${LATEST_VERSION//-alpha/~alpha}"
-	LATEST_VERSION="${LATEST_VERSION//-beta/~beta}"
+	for suffix in "rc" "alpha" "beta"; do
+		LATEST_VERSION="$(sed -E "s/[-.]?(${suffix}[0-9]*)/~\1/ig" <<< "$LATEST_VERSION")"
```

Comment:
Making the `sed` substitution here case insensitive catches a couple extra pre-release variants.
Unfortunately it looks like `~beta` is counted as later than `~RC` though.
See: https://github.com/OpenTTD/OpenTTD/tags